.yarn/versions/fa98e249.yml

```diff
@@ -0,0 +1,5 @@
+releases:
```

Comment:
I did my best here per the docs, but as a new contributor, I'm not sure exactly what all my change will affect and thus which packages need to be released.  Help here would be appreciated.

Reply:
In this particular case we only need to re-release `core` (because it has new code) and `cli` (to rebuild the binary), all others should be flagged as `declined` in the interface.

Reply:
Gotcha, I updated that